language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -515,13 +506,11 @@ renderInterfaceDef InterfaceDef{ifName, ifChoices, ifModule,
       , ifaceDefIface ifName Nothing ifChoices
       , [ "export declare const " <> ifName <> ":"
         , "  damlTypes.InterfaceCompanion<" <> ifName <> ", undefined, '" <> ifaceId <> "'> &"
-        , "  damlTypes.FromTemplate<" <> ifName <> ", " <> retroImplsIntersection <> "> &"
+        , "  damlTypes.FromTemplate<" <> ifName <> ", unknown> &"
```

Comment:
Does FromTemplate serve any other role than to declare the retroactive instances?

Reply:
Oh right, the doc is pretty explicit about it indeed. Thanks!